code_generation/data/attribute_classes/input.json

```diff
@@ -127,6 +127,55 @@
                 }
             ]
         },
+        {
+            "name": "AsymLineInput",
+            "base": "BranchInput",
+            "attributes": [
+                {
+                    "data_type": "double",
+                    "names": [
+                        "r_aa",
+                        "r_ba",
+                        "r_bb",
+                        "r_ca",
+                        "r_cb",
+                        "r_cc",
```

Comment:
OpenDSS uses low triangle, as many power system conventions to specify the matrix. Let's keep that way.

Reply:
i'm OK with that. the main reason why i raised the question is, because i saw the difference with `uk_12`, `uk_13`, `uk_23` in the 3w-transformer. While that case is not quite the same as this one, because uk is not an actual matrix (although it can be interpreted as such), it's a deviation from the current conventions and we should make such changes deliberately.

Closing this issue now cfr. outcome of the chat